images/win/scripts/Tests/Shell.Tests.ps1

```diff
@@ -0,0 +1,21 @@
+Describe "Shell" {
+    $IsWin16 = Test-IsWin16
+
+    $shellTestCases = @(
```

Comment:
Any reason why we can't add all 3 shells to this array and test?
I think `wslbash` can be added withoout any problem

Reply:
```
$shellTestCases = ()
if (Test-IsWin16) {
  $shellTestCases += @{}
}
```

Reply:
Thank you. Fixed.